### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by davidnuzik**
This PR adds a "Storage" page in the docs for k3s.

- Normalize page weights; order was not modified this just makes it easier to work with in the future.
- Add a storage page and content for the local path provisoner (built in to k3s) and longhorn.

Closes https://github.com/rancher/k3s/issues/863

### CODE REVIEW

File: content/k3s/latest/en/storage/_index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+k3s supports [Longhorn](https://github.com/longhorn/longhorn). Below we cover a simple example. For more information please reference the official documentation [here](https://github.com/longhorn/longhorn/blob/master/README.md).
+
+Apply the longhorn.yaml to install Longhorn.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn.yaml
+```
+
+Longhorn will be installed in the namespace `longhorn-system`
```
Comment (catherineluse) [inline content/k3s/latest/en/storage/_index.md]: add period

File: content/k3s/latest/en/storage/_index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      mountPath: /data
+    ports:
+    - containerPort: 80
+  volumes:
+  - name: volv
+    persistentVolumeClaim:
+      claimName: longhorn-volv-pvc
+```
+
+Confirm the pv and pvc are created. `kubectl get pv` and `kubectl get pvc` The status should be Bound for each.
```
Comment (catherineluse) [inline content/k3s/latest/en/storage/_index.md]: I would capitalize PV and PVC in sentences

File: content/k3s/latest/en/storage/_index.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+title: "Volumes and Storage"
+weight: 30
+---
+
+When deploying an application that needs to retain data, you’ll need to create persistent storage. Persistent storage allows you to store application data external from the pod running your application. This storage practice allows you to maintain application data, even if the application’s pod fails.
+
+# Local Storage Provider
+k3s comes with Rancher's Local Path Provisioner and this enables the ability to create persistent volume claims out of the box using local storage on the respective node. Below we cover a simple example. For more information please reference the official documentation [here](https://github.com/rancher/local-path-provisioner/blob/master/README.md#usage).
+
+Create a hostPath backed persistent volume and a pod to utilize it:
```
[the remark this answers is not in the document]
Comment (davidnuzik) [reply content/k3s/latest/en/storage/_index.md]: Reference https://github.com/rancher/local-path-provisioner#usage it should be PV not PVC by my understanding.

File: content/k3s/latest/en/storage/_index.md (hunk 1 above)
Comment (davidnuzik) [reply content/k3s/latest/en/storage/_index.md]: Fixed in incoming commit. thanks

File: content/k3s/latest/en/storage/_index.md (hunk 2 above)
Comment (davidnuzik) [reply content/k3s/latest/en/storage/_index.md]: Fixed in all areas in incoming commit.

File: content/k3s/latest/en/storage/_index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidnuzik) [reply content/k3s/latest/en/storage/_index.md]: As we discussed going with adding **claim** for now; seems like you are indeed correct. We can fix with a quick pr if needed if this is not the case.

**Pull request merged**